compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/plugins/CompilerPlugins.java

```diff
@@ -100,7 +100,7 @@ private static Class<?> loadPluginClass(String pluginClassName, ClassLoader clas
     private static ClassLoader createClassLoader(List<Path> jarDependencyPaths) {
         return AccessController.doPrivileged(
                 (PrivilegedAction<URLClassLoader>) () -> new URLClassLoader(getJarURLS(jarDependencyPaths),
-                        CompilerPlugins.class.getClassLoader())
+                        Thread.currentThread().getContextClassLoader())
```

Comment:
Did we decide to change this during the discussion as it could break something? Sorry, I couldn't remember.